terraform/projects/pr-dashboard/rds.tf

```diff
@@ -0,0 +1,103 @@
+resource "aws_db_subnet_group" "rds" {       
+  name = "default-db"                        
+  description = "default subnets"            
+  subnet_ids = ["${aws_subnet_wpt_id}"]
+}
+
+resource "aws_subnet_wpt" "wpt-subnet" {
```

Comment:
You can remove this, we have subnets defined for this project already in `network.tf`. The resources on a per-project basis are available cross-file.

Reply:
Done. A lot of this was copied from the internal bocoup rds.tf file.